src/App.js

```diff
@@ -1,10 +1,186 @@
-import React from 'react';
+//Ik ben gekomen tot opdracht 3 en heb componenten gemaakt voor de reset- en verzend button en de +/-counter
+
+import React, {useState} from 'react';
+import Button from "./components/Button/Button";
+import Counter from "./components/Counter/Counter";
 import './App.css';
 
 function App() {
-  return (
-    <>
-      <h1>Fruitmand bezorgservice</h1>
+    const [ strawberries, setStrawberries ] = useState( 0);
+    const [ bananas, setBananas ] = useState( 0);
+    const [ apples, setApples ] = useState( 0);
+    const [ kiwis, setKiwis ] = useState( 0);
+    function reset() {
+        setStrawberries(0);
+        setBananas(0);
+        setApples(0);
+        setKiwis(0);
+    }
+
+    const [firstNameValue, setFirstNameValue] = React.useState('');
+    const [lastNameValue, setLastNameValue] = React.useState('');
+    const [ageValue, setAgeValue] = React.useState('0');
+    const [zipValue, setZipValue] = React.useState('');
+    const [frequency, toggleFrequency] = React.useState('week');
+    const [timeslot, toggleTimeslot] = React.useState('Overdag')
+    const [remarksValue, setRemarksValue] = React.useState('');
+    const [termsConditions, toggleTermsConditions] = React.useState(false);
+    //const handleChange = () => { toggleTermsConditions(!termsConditions);};] <-- Vraag: Hoe kan de checkbox ook weer uitvinken zonder deze statement?
+    const handleSubmit = (e) => {
+        e.preventDefault();
+        console.log(
+            `Aantal aardbeien: ${strawberries}
+            Aantal bananen: ${bananas}
+            Aantal appels: ${apples}
+            Aantal kiwi's: ${kiwis}
+            Voornaam: ${firstNameValue}
+            Achternaam: ${lastNameValue}
+            Leeftijd: ${ageValue}
+            Postcode: ${zipValue}
+            Bezorgfrequentie: ${frequency}
+            Tijdslot: ${timeslot}
+            Opmerking: ${remarksValue}
+            Akkoord: ${termsConditions}
+        `);
+    }
+
+        return (
+        <>
+            <h1>Fruitmand bezorgservice</h1>
+                <section>
+                    <aricle>
+                        <p>🍓 Aardbeien</p>
+                        <Counter
+                            fruitCount={strawberries}
+                            setFruitCount={setStrawberries}
+                        />
+
+                    </aricle>
```

Comment:
Ik zie dat je een Counter component hebt gemaakt, en dat is zeker goed! Ik zie alleen wel dat wanneer dat component gebruikt wordt, er dus altijd nog een article omheen gezet wordt met een paragraph over welk fruitsoort het gaat. Deze dubbeling zie ik dus meerdere keren terugkomen. Ik had het zelf dan bínnen het Counter component geplaatst (en dan wellicht ook een andere naam gegeven 😉). Dan heb je nóg minder code dubbeling